content/post/leadership-ci.md

```diff
@@ -0,0 +1,118 @@
++++
+date = "2019-04-06T8:09:26-07:00"
+title = "Leadership CI"
+author = "Jessica Frazelle"
+description = "Questions to ask yourself to make sure you are in the right mindset to think and behave as a leader."
++++
+
+This post is co-authored by [Kathy Simpson](https://github.com/simpsoka).
+
+> “understanding the true nature of instinctive decision making requires us to be forgiving of those people trapped in circumstances where good judgment is imperiled.” 
+> ― Malcolm Gladwell, Blink: The Power of Thinking Without Thinking
+
+
+As leaders, being consistent in our actions even when under pressure is of
+utmost importance. When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
```

Comment:
Hmmm, consistency isn’t always good unless it’s consistent integrity..... we want to have room to grow and change our minds - there’s a question for this, too. So maybe something that gives leaders a little room, while still suggesting a structure to move around in. 

Maybe like..... 

As leaders, setting up a structure that helps us navigate decisions under pressure is of the utmost importance.